integration-tests/gradle/projects/it-js-ir-0/src/main/kotlin/RootPackageClass.kt

```diff
@@ -18,4 +20,6 @@ fun test(list: MutableList<Int>) = "list"
 
 @JsModule("is-sorted")
 @JsNonModule
-external fun <T> sorted(a: Array<T>): Boolean
\ No newline at end of file
+external fun <T> sorted(a: Array<T>): Boolean
+
+external interface TextLinkProps: AnchorHTMLAttributes<HTMLAnchorElement>
```

Comment:
If something were to break again, this would catch it, right? Not a JS expert, so I fail to understand why - is it considered to be a dynamic type or something? 

Maybe leaving a comment there would not hurt, so that other people do not remove it. i.e
```
// do not remove, this declaration is used to check presence of dynamic type deserializer
```